console/src/main/webapp/manager/app/components/users/users.es6

```diff
@@ -81,32 +81,40 @@ class UsersController {
       : true
   }
 
-  exportCSV () {
-    const download = this.$injector.get('ExportCSV')
+  export_ (fileType) {
+    const download = this.$injector.get(`Export${fileType.toUpperCase()}`)
     download(this.selection).then(result => {
       if (result.status !== 200) {
         throw new Error(`Cannot fetch users list. error ${result.status}`)
       }
-      const blob = new Blob([result.data], { type: 'text/csv' })
-      window.open(window.URL.createObjectURL(blob))
+      let mimetype = ''
+      switch (fileType) {
+        case 'vcf':
+          mimetype = 'text/x-vcard'
+          break
+        default:
+          mimetype = `text/${fileType}`
+      }
+      const blob = new Blob([result.data], { type: mimetype })
+      const a = document.createElement('a')
+      a.href = window.URL.createObjectURL(blob)
+      a.target = '_blank'
+      a.download = `users.${fileType}`
```

Comment:
Could you generate a filename with a timestamp like `20190723-100356_users_export.[csv|vcf]` ?